src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -13,21 +14,50 @@ import {
 import { getDictionaryText, Path } from 'shared/utils';
 import { ItemResponseType } from 'shared/consts';
 
-import { ActivityFormValues } from './BuilderApplet.types';
+import { ActivityFormValues, ItemFormValues } from './BuilderApplet.types';
 
 const { t } = i18n;
 
 export const isAppletRoute = (path: string) => matchPath(`${page.builderApplet}/*`, path);
 
+export const getNewActivityItem = (item?: ItemFormValues) => ({
+  responseType: '',
+  name: '',
+  question: '',
+  config: {},
+  isHidden: false,
+  ...item,
+  id: undefined,
+  key: uuidv4(),
+  ...((item?.responseType === ItemResponseType.SingleSelection ||
+    item?.responseType === ItemResponseType.MultipleSelection) && {
+    responseValues: {
+      ...item.responseValues,
+      options: (item.responseValues as SingleAndMultipleSelectItemResponseValues)?.options?.map(
+        (option: SingleAndMultipleSelectionOption) => ({
+          ...option,
+          id: uuidv4(),
```

Comment:
How do you think, maybe it's better to use key for new option?
Because on update we need to send id in payload.
![image](https://user-images.githubusercontent.com/116339462/232700932-66df08a4-5232-482e-a2f3-287f75d6c294.png)